style.css

```diff
@@ -1,299 +1,328 @@
-    max-width:129.6rem;
-    margin:0 auto;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    margin-top: 2rem;
-}
-.nav-menu-box{
-    width: 49.5%;
-    height: fit-content;
-    background-color: transparent;
-    overflow: hidden;
-}
-.menu-box{
-    padding: 1.6rem;
-}
-.menu-title{
-    margin-bottom: 2rem;
+  transform: translateY(100%);
+  transition: all 0.4s linear;
+  opacity: 0;
+  visibility: hidden;
+  pointer-events: none;
+}
+.active .menu-wrapper {
+  transform: translateY(0);
+  opacity: 1;
+  visibility: visible;
+}
+.menu-container {
+  max-width: 129.6rem;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-top: 2rem;
+}
+.nav-menu-box {
+  width: 49.5%;
+  height: fit-content;
+  background-color: transparent;
+  overflow: hidden;
+}
+.menu-box {
+  padding: 1.6rem;
+}
+.menu-title {
+  margin-bottom: 2rem;
+}
+.menu-title hr {
+  margin-top: 0.6rem;
+}
+.category-box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 1.5%;
+}
+.category {
+  width: 49%;
+  border: 1px solid #dddddd;
+  box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  border-radius: 1rem;
+  margin-bottom: 1%;
+  background-color: #ffffff;
+}
+.category-name {
+  display: flex;
+  gap: 2rem;
+  padding: 4%;
+}
+.category-name a {
+  text-decoration: none;
+  color: black;
+}
+.category-name a:nth-child(2) {
+  color: rgb(63, 62, 62);
+  font-size: 0.9rem;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+.category-name img {
+  background-color: black;
 }
-.menu-title hr{
-    margin-top: 0.6rem;
-}
-.category-box{
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    gap: 1.5%;
-}
-.category{
-    width: 49%;
-    border: 1px solid #DDDDDD;
-    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
-    border-radius: 1rem;
-    margin-bottom: 1%;
-    background-color: #FFFFFF;
-}
-.category-name{
-    display: flex;
-    gap: 2rem;
-    padding: 4%;
+/*=== XX === Menu Bar code ends ==== XX ===*/
+
+/*======= Footer Section code start========*/
+.footer-container {
+  background-color: #f0f0e5;
+  margin-top: 10vh;
+  /* I would recommend using this font for the whole site */
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  /* Centering items inside the footer */
+  justify-content: center;
+  align-items: center;
 }
-.category-name a{
-    text-decoration: none;
-    color: black;
+
+footer {
+  width: 100vw;
+  /* background-color: grey; */
+  font-size: larger;
+  padding: 3.2rem;
+  text-align: center;
 }
-.category-name a:nth-child(2){
-    color: rgb(63, 62, 62);
-    font-size: 0.9rem;
-    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif ;
+
+#footer-logo {
+  /* Setting the size for the logo */
+  width: 10vw;
```

Comment:
fix logo width in responsive mobile screen size. logo is decreasing its width wen we shift to mobile size